spec/models/post_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let (:author) { User.new(:id=>1,:name => "Dulce", :lastname => 'Woof', :email => "dulce@woof.com", :password => "123456", :password_confirmation => "123456") }
+  subject { Post.new }
+  
+  describe 'basic validations for post model' do
+    it 'is not valid without content' do
+      expect(subject).not_to be_valid
+    end
+
+    it 'is not valid if it has more than 500 char' do
+      subject.content = 'a' * 501
+      expect(subject).not_to be_valid
+    end
+
+    it 'is not valid if it has proper length data but no author' do
+      subject.content = 'a' * 499
+      expect(subject).not_to be_valid
+    end
+
+    it 'is valid if it has proper length data AND author' do
+      subject { Post.new }
+      subject.content = 'a' * 499
+      subject.author = author
+      expect(subject).to be_valid
+    end
+  end
+
+  describe 'validating correct associations' do
```

Comment:
This is a good opportunity to use your `shoulda` matchers. Take a look at the gem's repository and look at the examples there to get an idea on how to incorporate the syntax in here. I see you have it for User but not for comments or likes. Althought comments or likes is outside the scope of this review.

[OPTIONAL]